tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -705,3 +705,148 @@ def _find_output(predictions):
                 'If you need to use another layer, please specify it explicitly'
             )
         return filter_outputs[0]
+
+
+class FaceBoxesSSDDecoder(Adapter):
```

Comment:
please register your adapter in __init__.py

Reply:
and add its description in readme.md

Reply:
Adapter has been added in init.py

Reply:
Description has been added